### PR TITLE
feat: add transaction activity

### PR DESCRIPTION
**Pull request by ethan-tbd**
Closes https://github.com/TBD54566975/didpay/issues/35

Keeping all widgets within `HomePage` to reduce clutter. Below screen recording shows what the `ListView` looks like when populated with many items.

Simulator Screenshot - iPhone 15 Pro Max - 2024-01-31 at 16 57 06

Simulator Screenshot - iPhone 15 Pro Max - 2024-01-31 at 16 57 12

https://github.com/TBD54566975/didpay/assets/125412902/ea89ad29-ff84-443a-8ada-00f72c5536a7

https://github.com/TBD54566975/didpay/assets/125412902/be0aab13-d597-46f4-bc67-4e29a9ca3b99

### CODE REVIEW

File: frontend/lib/features/home/home_page.dart [hunk 1]
```diff
@@ -1,21 +1,169 @@
...
+              padding: const EdgeInsets.symmetric(
+                  horizontal: Grid.side, vertical: Grid.xs),
+              child: Text(
+                Loc.of(context).activity,
+                style: Theme.of(context).textTheme.bodyMedium,
+              ),
+            ),
+            Expanded(
+              child: txns.isEmpty
+                  ? buildEmptyState(context)
```
Comment (wesbillman) [inline frontend/lib/features/home/home_page.dart:32]: @ethan-tbd I wonder if we should center this in the empty space on the screen vs. at the top. Whatcha think?

File: frontend/lib/features/home/home_page.dart [hunk 2]
```diff
@@ -1,21 +1,169 @@
...
+                  : ListView(
```
Comment (wesbillman) [inline frontend/lib/features/home/home_page.dart:33]: We could move this to a func to make this scaffolding really easy to parse quickly

File: frontend/lib/features/home/home_page.dart [hunk 3]
```diff
@@ -1,21 +1,169 @@
...
+  }
+
+  Widget buildAccountBalance(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: Grid.side),
+      child: Container(
+        decoration: BoxDecoration(
+          border: Border.all(
+              color: Theme.of(context).colorScheme.outline, width: 2.0),
+          borderRadius: BorderRadius.circular(25.0),
```
Comment (wesbillman) [inline frontend/lib/features/home/home_page.dart:86]: Do we want a standard radius for these? We could use the grid or define a similar radius size somewhere.

File: frontend/test/features/home/home_page_test.dart [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 import 'package:flutter_starter/features/deposit/deposit_page.dart';
+import 'package:flutter_starter/features/home/home_page.dart';
 import 'package:flutter_starter/features/withdraw/withdraw_page.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 import '../../helpers/widget_helpers.dart';
 
 void main() {
-  group('AccountBalance', () {
+  group('HomePage', () {
```
Comment (wesbillman) [inline frontend/test/features/home/home_page_test.dart:9]: Thanks!

**Review by wesbillman — APPROVED** [review]

File: frontend/lib/features/home/home_page.dart (hunk 1 above)
Comment (ethan-tbd) [reply frontend/lib/features/home/home_page.dart:32]: setting `mainAxisAlignment` to center seems a bit too low? would we have to play with `Align` or just use a bigger`SizedBox`?

Simulator Screenshot - iPhone 15 Pro Max - 2024-01-31 at 17 21 07

Comment (ethan-tbd) [reply frontend/lib/features/home/home_page.dart:32]: @wesbillman ^ or do you think that's fine as is? been staring at it for too long to tell 😅

**Pull request merged**